docs/drivers/azure.md

```diff
@@ -10,51 +10,105 @@ parent="smn_machine_drivers"
 
 # Microsoft Azure
 
-Create machines on [Microsoft Azure](http://azure.microsoft.com/).
+You will need an Azure Subscription to use this Docker Machine driver.
+[Sign up for a free trial.][trial]
 
-You need to create a subscription with a cert. Run these commands and answer the questions:
+> **NOTE:** This documentation is for the new version of  Azure driver started
+> shipping with v0.7.0 and it is not backwards-compatible with the old Azure
+> driver. If you like to manage your existing Azure machines, please download
```

Comment:
If you want to continue managing your existing Azure machines, please download ...